DEV-gitSpy/gitSpyInit.js

```diff
@@ -1,26 +1,33 @@
 var fs = require('fs');
 var spawnSync = require('child_process').spawnSync;
 
+var spyUpdate = '/usr/local/bin/spyupdate';
+var preCommit = './.git/hooks/pre-commit';
 var postReWritePath = './.git/hooks/post-rewrite';
```

Comment:
The pre-rebase hook actually prevented me from rebasing when my gitSpy keys were not set up yet. I think we should take this out since we have the "spyupdate" command that works every time!